### PR TITLE
Prepare for First Public Working Draft

### PR DESCRIPTION
**Pull request by marcoscaceres**
Prepares document for FPWD. 

Did a big cleanup. Removed dependencies on WebIDL, which don't make sense in a store context. 

Moved the categories into the document itself, as this is a living "registry". 

Removed processing of categories, as this is not targeted as user agents. 

Removed Rob as retired editor, as we still have Rob as editor of the manifest spec - but didn't contribute to this spec. Acknowledged his contribution more fully in acknowledgement section.

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@aarongustafson, once this lands, I'll start a CFC in WebAppsWG to publish as a Note.

File: index.html [hunk 1]
```diff
@@ -31,18 +23,27 @@
...
-        This specification defines supplementary members of the Web App
-        Manifest that provide additional metadata related to how the web
-        application may be presented in the context of a digital storefront,
-        installation dialog, or other surfaces where this web application may
-        be marketed or distributed.
+        This document is a registry of supplementary members for the
+        [[[appmanifest]]] specification that provide additional metadata for
+        use in a digital storefront or other surfaces where this web
+        application may be marketed or distributed (i.e., outside the context
+        of a web browser).
```
Comment (aarongustafson) [inline index.html:37]: I don't think we need to be this prescriptive here. I know we, at least, are open to/experimenting with enhancing the installation experience with some of this data.

File: index.html [hunk 2]
```diff
@@ -212,53 +222,44 @@ <h3>
       </section>
       <section>
         <h3>
-          <code title="">screenshots</code> member
+          `screenshot` member
```
Comment (aarongustafson) [inline index.html:225]: `screenshots`

File: index.html [hunk 3]
```diff
@@ -212,53 +222,44 @@ <h3>
...
-          <dfn>platform</dfn> member
+          `platform` member
         </h3>
         <p>
-          The {{ScreenshotImageResource/platform}} member represents the
-          <a>platform</a> for which a given screenshot applies. Authors are
-          encouraged to only use this member when the screenshot is only
-          applicable in a specific context.
+          The <code><dfn>platform</dfn></code> member is a <a>string</a> that
+          represents the operating system for which a given screenshot applies.
```
Comment (aarongustafson) [inline index.html:260]: Do you think we should forego the wiki and add these here as well (like you did with categories)?

File: index.html [hunk 4]
```diff
@@ -269,27 +270,30 @@ <h3>
...
-    <section id="issue-summary"></section>
-    <section id="index"></section>
+    <section class="appendix">
+      <h2>
+        Acknowledgements
+      </h2>
+      <p>
+        Rob Dolin for spearheading a lot of this work, specially the
+        `iarc_rating_id` member, and Kenneth Rohde Christiansen for helping to
+        vet these idea.
```
Comment (aarongustafson) [inline index.html:291]: Ideas

File: index.html (hunk 3 above)
Comment (marcoscaceres) [reply index.html:260]: yes, we should add them here.

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html:37]: ok, will split this up a bit and add back the install aspect.

**Comment by marcoscaceres** [comment]
I exported "apply" from Web Manifest earlier today, so it's not linking right now... usually takes a few hours for Shepherd to pick up the exports.

**Pull request merged**